### PR TITLE
feat(eslint-plugin-rynair): ESLint Ryanair plugin publish

### PR DESCRIPTION
**Pull request by patrykgaluszka**
ESLint Ryanair plugin publish including first 7 custom rules regarding mostly AngularJS 1.7
migration.

BREAKING CHANGE: Packages folder names are changed, because of new systematic

### CODE REVIEW

File: packages/eslint-plugin/lib/helpers/angularJs.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * @description Gets configuration object from directive registration,
+ * returned in a factory function
+ * @param {ASTNode} node (CallExpression)
+ * @returns {ASTNode} (ObjectExpression)
+ */
+const getDirectiveConfig = R.pipe(
+  R.path(['arguments', '1']),
+  R.pathOr([], ['body', 'body']),
+  R.find(el => el.type === 'ReturnStatement'),
+  R.prop('argument'),
```
Comment (maksymddd) [inline packages/eslint-plugin/lib/helpers/angularJs.js]: trailing comma

Comment (patrykgaluszka) [reply packages/eslint-plugin/lib/helpers/angularJs.js]: Done

File: packages/eslint-plugin/lib/helpers/angularJs.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/**
+ * @fileoverview Helper functions specific for AngularJS (version 1.X)
+ */
+
+const R = require('ramda');
```
[the remark this answers is not in the document]
Comment (patrykgaluszka) [reply packages/eslint-plugin/lib/helpers/angularJs.js]: I could, but I am not found of that idea. I think it's clearer while you see the context (R.). It's short enough to keep code clear and elegant. Ramda functions are sometimes named same as prototype methods: `map`, `filter`, `find` and in my opinion it's good to know you are using Ramda. I am destructuring more specific named functions, so overriding the name or mismatching it is not an real issue (ie: `findCalleeNamed`, `getFinalCallee`).

[the remark this answers is not in the document]
Comment (izifortune) [reply packages/eslint-plugin/lib/helpers/angularJs.js]: prototype methods will not be touched and importing only the specific one gives you an idea of what methods are used in the first place.

**Comment by patrykgaluszka** [comment]
@izifortune Changes applied

**Pull request merged**